lib/chat/messages.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+
+const dummyData = [
+  {
+    "name": "Lydia",
+    "color": Colors.blue,
+    "lastMessage": "What's that snap",
+    "isSent": true,
+  },
+  {
+    "name": "Ashley",
+    "color": Colors.purple,
+    "lastMessage": "Hey there 😛",
+    "isSent": true,
+  },
+  {
+    "name": "Mike",
+    "color": Colors.green,
+    "lastMessage": "I'm taking that as a yes",
+    "isSent": true,
+  },
+  {
+    "name": "Hawk",
+    "color": Colors.orange,
+    "lastMessage": "U can get this dick",
+    "isSent": true,
+  },
+  {
+    "name": "Pat",
+    "color": Colors.yellow,
+    "lastMessage": "Your welcome handsome! I'm fine just doing some laundry!",
+    "isSent": false,
+  },
+  {
+    "name": "Maweenie",
+    "color": Colors.red,
+    "lastMessage": "Damn that's crazy",
+    "isSent": false,
+  },
+  {
+    "name": "Simp",
+    "color": Colors.indigo,
+    "lastMessage": "wow",
+    "isSent": false,
+  },
+];
+
+class Messages extends StatelessWidget {
+  Messages({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        Text(
```

Comment:
Similarly to the comment made for the `New Matches` text, you can add the `Padding` widget to add quick spacing before the `Messages` text.

```dart
Padding(
   padding: const EdgeInsets.only(top: 20, bottom: 10), 
   child: Text("Messages", 
   textAlign: TextAlign.left,
      style: GoogleFonts.muli(
      textStyle: TextStyle(
      color: Colors.black, 
      letterSpacing: .5, 
      fontSize: 18.0, 
      fontWeight: FontWeight.bold
   )
)
```

With padding added to the `New Matches` and `Messages` headers, it looks like this:
> <img src="https://user-images.githubusercontent.com/23146829/82512044-29972a80-9acc-11ea-89e2-9d8dcc54e8b8.png" alt="Screenshot of NewMatches page with padding added to the headers" width=200 />